### PR TITLE
Rename capsule to proxy

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: bats/katello-bats [hunk 1]
```diff
@@ -34,8 +34,8 @@ for arg do
       swapfile)
         swapfile=true
         ;;
-      capsule)
-        katello_capsule=true
+      proxy)
+        katello_proxy=true
```
Comment (ehelms) [inline bats/katello-bats]: Nitpick -- should we stick to `proxy_content` for consistency across all the re-names?

File: playbooks/roles/foreman_proxy_content/tasks/koji_repos.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (ehelms) [inline playbooks/roles/foreman_proxy_content/tasks/koji_repos.yml]: Would an install not be able to use the `katello_repositories` role to achieve this?

Comment (stbenjam) [inline playbooks/roles/foreman_proxy_content/tasks/koji_repos.yml]: This is just a straight rename, it's there now for the capsule module.

Comment (stbenjam) [inline playbooks/roles/foreman_proxy_content/tasks/koji_repos.yml]: I'd rather not get into the business of trying to refactor things alongside the rename, we could fix this after.

File: bats/katello-bats (hunk 1 above)
Comment (ekohl) [inline bats/katello-bats]: I think this case should just be proxy, in line with the filename. On the other hand I'd also like to kill this file (like https://github.com/Katello/forklift/pull/334 helps to do)

Comment (ehelms) [inline bats/katello-bats]: Works for me

File: playbooks/roles/foreman_proxy_content/tasks/koji_repos.yml (hunk 2 above)
Comment (ehelms) [inline playbooks/roles/foreman_proxy_content/tasks/koji_repos.yml]: Touche -- I didn't put two and two together (equals 5 right?)

File: bats/katello-bats (hunk 1 above)
Comment (ehelms) [inline bats/katello-bats]: @stbenjam thoughts on this?

Comment (stbenjam) [inline bats/katello-bats]: Oh I thought we could keep it katello_proxy for now, it's going to go anyway with #334.

Comment (ehelms) [inline bats/katello-bats]: Fair enough

**Pull request merged**